### PR TITLE
[DISCOVERY-130] - system memory for vcenter scans

### PR DESCRIPTION
**Pull request by bruno-fs**
- Add `system_memory_bytes` column to system fingerprints
- collect this fingerprint for vcenter scans
- refactor fingerprint/tasks.py to better accommodate this change

### CODE REVIEW

**Comment by bruno-fs** [comment]
Will keep as a draft until #2179 is merged

File: quipucords/fingerprinter/formatters.py [hunk 1]
```diff
@@ -15,3 +15,13 @@ def format_mac_addresses(mac_addresses):
     if isinstance(mac_addresses, list):
         mac_addresses = list(map(lambda x: x.lower(), mac_addresses))
     return mac_addresses
+
```
Comment (nicolearagao) [inline quipucords/fingerprinter/formatters.py:18]: That approach will simplify Satellite code a lot.

**Review by nicolearagao — APPROVED** [review]
LGTM 👍🏼

**Review by gtanzillo — APPROVED** [review]
LGTM 👍

**Pull request merged**